content/en/docs/Contribution guidelines/coming-soon.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
While the `coming-soon` doc made sense in a git book, I don't think it's doing what we intended it to do well enough for the doc site. For one, it appears as a sub tab under the contributing guideline tab. We can also just not have the `coming soon` doc, and instead have no links at all. Since we already have warnings everywhere that this site is under construction, it should be fine. 
So for example instead of 
```For more information on approving manual updates to operators, please see [How do I approve an update?](/docs/contribution-guidelines/coming-soon/)```

we just do 
```For more information on approving manual updates to operators, please see How do I approve an update?```

Reply:
Agreed. But by doing so we will lose the fact that this needs to be revisited. Do I add a note right below it that the document and link needs to be updated?

Reply:
@harishsurf that was okay when we were still in the building stage, but since it's being showcased in the summit now and we're live, I think we just to make a TODO list elsewhere (eg JIRA) instead of right on the site.